grobid-core/src/main/java/org/grobid/core/sax/PDFALTOOutlineSaxHandler.java

```diff
@@ -92,10 +92,22 @@ public void startElement(String namespaceURI, String localName,
 			nodes.put(currentId,currentNode);
 			if (currentParentId != -1) {
 				DocumentNode father = nodes.get(currentParentId);
+                // Correction proposed by Luc Bougé @lucbouge
```

Comment:
You don't need all these comments, your name will be registered anyway in the git history